### PR TITLE
doc: Add italiano translation

### PR DESCRIPTION
**Pull request by JoeZijunZhou**
@Atena1118

### CODE REVIEW

**Comment by JoeZijunZhou** [comment]
@sciulli

File: README_IT.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+Translation: *italiano*
```
Comment (CNYoki) [inline README_IT.md]: [简体中文](./README.md) | [English](./README_EN.md) | [日本語](./README_JP.md) | Italiano

Comment (CNYoki) [reply README_IT.md]: [简体中文](./README.md) | [English](./README_EN.md) | [日本語](./README_JP.md) | Italiano

File: README_IT.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+Nei rigaurdi dell'esplosione in Cina della polmonite provocato dal Coronavirus nuovo, questo progetto tende allo scopo di raccogliere le informazioni su ospedali, alberghi, fabbriche, logistiche, donazioni, fondi, prevenzioni, rimedi e notizie, e allo scopo di facilitare lo scaimbio informativo e l'allocazione efficace di risorse sociale. 
+
+Questo è il deposito principale, l'importazione di tutti i dati viene fatta automaticamente dallo script,  **_non consegnare direttamente i dati a questo deposito_**, per ulteriori informazioni, per favore leggi questo documento.
+
+## Collaborazione
+
+Per la nostra meglio collaborazione, si stabilisce il seguente processo 
+
+![img](http://www.plantuml.com/plantuml/png/RP31Jkf068NtynIJkMiImf85uQxGdT4d6DfH6akRj5EDEqb4H2MO420HerOn4arQZT5e0NcPcIckU0NR3bqOtJKzttyotodQ55lKgUg0QbGdSDUfO2ENpMKXRxNPz4AyriBH2G1OeQO57PjODiGsHABx95gUQ9-npy5ylxwO7B7nc4sxB0WMaoQ2_zQ92XHJrub2DTEmeLtHgcPo6bwzy9kHw3M4UukMnTXHDPgat7F5zJkVzSN1B2gEcaeM8GPGCSLbR1EufT6AKqxOaaPNea_v5ZRkyA23036eHlTW6IlRn50Jxl_QAjmWrWwnqhgKshHCWwOORxR2H__B_GW7tjz2G0wGAKYTF4HivegQ7-yG316G6fbVUMpaNI8WHuXpQH41Cf8Ozyv5_stUUE378-vFUFqE0I39-2XrogVpIrwIop_n0gbwfY3zVfoq_Vdz8J_jyUTkE0mGA4QfKzM_0G00)
```
Comment (CNYoki) [inline README_IT.md]: link换成https://yokii.cn/i/en.jpg

Comment (JoeZijunZhou) [reply README_IT.md]: 好的

File: README_IT.md (hunk 1 above)
Comment (JoeZijunZhou) [reply README_IT.md]: 好的

**Pull request merged**